watertap/flowsheets/full_water_resource_recovery_facility/BSM2.py

```diff
@@ -526,35 +526,34 @@ def setup_optimization(m, reactor_volume_equalities=False):
     add_effluent_violations(m)
 
 
-def add_effluent_violations(m):
-    # TODO: update "m" to blk; change ref to m.fs.Treated instead of CL1 effluent
```

Comment:
I haven't been able to figure out why, but this change is breaking the BSM2 GUI, so I've reverted it in #1491